libs/hdf-converters/src/cyclonedx-sbom-mapper.ts

```diff
@@ -0,0 +1,523 @@
+            } else if (input.authors) {
+              // Join list of component authors
+              let msg = '';
+              for (const author of input.authors as Record<string, unknown>[]) {
+                msg += `${author.name}, `;
+              }
+              return msg.slice(0, -2);
+            } else if (input.manufacturer) {
+              // If we can't pinpoint the exact authors, resort to the organization
+              return `${(input.manufacturer as Record<string, unknown>).name}`;
+            } else {
+              return undefined;
+            }
+          }
+        },
+        summary: {
+          path: 'raw.metadata.component',
+          transformer: (input: Component): string | undefined =>
+            input.description ? `${input.description}` : undefined
+        },
+        copyright: {
+          path: 'raw.metadata.component',
+          transformer: (input: Component): string | undefined =>
+            input.copyright ? `${input.copyright}` : undefined
+        },
+        license: {
+          path: 'raw.metadata.component',
+          transformer: (input: Component): string | undefined => {
+            let message = '';
+            if (Array.isArray(input.licenses)) {
+              // Join together all applicable licenses for this component
+              input.licenses.map((license) => {
+                message = message.concat(`${license.license.id}, `);
+              });
+              return message.slice(0, -2);
+            }
+            // If there are no found licenses, remove field
+            return undefined;
+          }
+        },
+        supports: [],
+        attributes: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'vulnerabilities',
+            key: 'id',
+            tags: {
+              nist: {
+                path: 'cwes',
+                transformer: getNISTTags
+              },
+              cci: {
+                path: 'cwes',
+                transformer: (input: CweRepository): string[] =>
+                  getCCIsForNISTTags(getNISTTags(input))
+              },
+              cwe: {path: 'cwes', transformer: formatCWETags}
+            },
+            descriptions: [
+              {
+                path: 'detail',
+                transformer: (
+                  input: Record<string, unknown>
+                ): Record<string, unknown> | undefined =>
+                  input ? {data: input, label: 'Detail'} : undefined
+              } as unknown as ExecJSON.ControlDescription,
+              {
+                path: 'recommendation',
+                transformer: (
+                  input: Record<string, unknown>
+                ): Record<string, unknown> | undefined =>
+                  input ? {data: input, label: 'Recommendation'} : undefined
+              } as unknown as ExecJSON.ControlDescription,
+              {
+                path: 'workaround',
```

Comment:
Fix text 2? Combine w/ 1?

Reply:
`Workaround` describe temporary solutions that reduce the likelihood of occurrence/impact. Doesn't seem too appropriate for a fix text (i.e., misleading).

Reply:
Concat to fix.